r/R/pseudobulk_differential_expression.R

```diff
@@ -0,0 +1,34 @@
+#' Run Pseudobulk Differential Expression
+#'
+#' @param pbulk a SeuratObject with counts aggregated by sample for one cluster.
+#'
+#' @return a data.frame of differential expression results.
+#' @export
+#'
+runPseudobulkDE <- function(pbulk) {
+    group <- pbulk$custom
+
+    y <- edgeR::DGEList(
+        pbulk[['RNA']]@counts,
+        samples = data.frame(group),
+        genes = pbulk@misc$gene_annotations[, "name", drop = FALSE])
+
+
+    y <- y[edgeR::filterByExpr(y, group=group),]
```

Comment:
@ogibson-biomage brought up that would be good to show filtered genes to user as well in case they search for specific gene that has been filtered.